src/libraries/System.Runtime.CompilerServices.Unsafe/src/ApiCompatBaseline.txt

```diff
@@ -0,0 +1,9 @@
+CannotRemoveAttribute : Attribute 'System.Diagnostics.CodeAnalysis.AllowNullAttribute' exists on parameter 'left' on member 'System.Runtime.CompilerServices.Unsafe.AreSame<T>(T, T)' in the contract but not the implementation.
```

Comment:
I added this to avoid having to add the attributes to the IL as it was verbose because we need to if def depending on the tfm to reference System.Private.CoreLib.